### PR TITLE
63 creation of the function that executes the areas

### LINKED ISSUE #63: creation of the function that executes the areas
## Describe the feature
creation of the function that will read the areas in the db. And realization of the algo that will work for all the user's areas

## TO DO
creation of the algorithm.

### PR DESCRIPTION
**Pull request by STom6**
# New feature
The area function, which performs reactions if the action trigger is triggered for all user areas contained in the db.
# Updated feature
Improvement of OpenMéteoService, return true or false.
Generalisation of the tweetAct & SendMail functions to use them as pointers to functions.
# Bug fixed

### CODE REVIEW

**Comment by STom6** [comment]
Screenshot 2023-02-05 at 09 23 52

Screenshot 2023-02-05 at 09 24 21

Screenshot 2023-02-05 at 11 01 27

Screenshot 2023-02-05 at 11 01 41

Screenshot 2023-02-05 at 09 24 51

Screenshot 2023-02-05 at 09 24 55

Screenshot 2023-02-05 at 09 25 00

Screenshot 2023-02-05 at 09 25 07

File: Server/Services/googleService.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = {
      * @param {*} userName Name of the issuer
      * @param {*} uid the user's uid
      */
-    send_mail: function(mailContent, subject, userName, uid) {
+    send_mail: function(subject, mailContent, uid, req, res) {
```
Comment (osvegn) [inline Server/Services/googleService.js:18]: Update param documentation

**Review by osvegn — CHANGES_REQUESTED** [review]

**Review by JohanCDev — DISMISSED** [review]
ntr

**Review by CedricCORGE — APPROVED** [review]
ntr

**Review by osvegn — APPROVED** [review]
Look good

**Pull request merged**